R/ui-legacy.R

```diff
@@ -13,8 +13,7 @@
 #'
 #'   usethis itself now uses cli internally for its UI, but these new functions
 #'   are not exported and presumably never will be. There is a developer-focused
-#'   article on the process of transitioning usethis's own UI to use cli (LINK
-#'   TO COME).
+#'   article on the process of transitioning usethis's own UI to use cli [here](https://cli.r-lib.org/articles/usethis-ui.html)
```

Comment:
Actually the intended link is to a usethis article:

https://usethis.r-lib.org/articles/ui-cli-conversion.html

When you update this PR can you also reword the sentence so that instead of having a "here" link, the hyperlink is attached to the article's title (Converting usethis's UI to use cli)? Thanks!